mssqlcli/i18n.py

```diff
@@ -0,0 +1,19 @@
+from importlib import import_module
+import gettext
+import os
+
+CLI_NAME = 'mssql-cli'
+MODULE = 'mssqlcli'
+
+def translate():
```

Comment:
Should we use gettext.install?
[https://docs.python.org/3/library/gettext.html#gettext.install](https://docs.python.org/3/library/gettext.html#gettext.install)

Reply:
The project is built and runs successfully without additional installation for gettext since it is built-in module already included in default python.

Reply:
Invoking `gettext.install()` will install the _() function to translate string. This way, you don't have to add the `_ = translate()` code in the modules needing translation.